sdk/ai/Azure.AI.Inference/samples/Sample8_ChatCompletionsWithTelemetry.md

```diff
@@ -0,0 +1,145 @@
+using Azure.Monitor.OpenTelemetry.Exporter;
+using OpenTelemetry.Metrics;
+```
+
+Now we will define the `endpoint`, `credential` and `model`. In this example we will take these values from the environment variables.
+```c#
+var endpoint = new Uri(System.Environment.GetEnvironmentVariable("MODEL_ENDPOINT"));
+var credential = new AzureKeyCredential(System.Environment.GetEnvironmentVariable("GITHUB_TOKEN"));
+var model = System.Environment.GetEnvironmentVariable("MODEL_NAME");
+```
+
+Next, we will define the constant, which defines, what [Activity](https://learn.microsoft.com/dotnet/api/system.diagnostics.activity) and [Meter](https://learn.microsoft.com/dotnet/api/system.diagnostics.metrics.meter) do we want to listen to. In Azure.AI.Inferencecase they are both named `Azure.AI.Inference.ChatCompletionsClient`.
+
+```c#
+const string ACTIVITY = "Azure.AI.Inference.ChatCompletionsClient";
+```
+
+To log metrics and events to Application Insights we will need to get the connection string. Please open the Azure portal, create the Application Insights resource you wish to use for storing the telemetry. After that open the main page and find the "Connection String". It generally will have the format similar to "InstrumentationKey=xxxxxxxx-xxxx-xxxx-xxxx-xxxxxxxxxxxx;IngestionEndpoint=https://region-x.in.applicationinsights.azure.com/;LiveEndpoint=https://eastus.livediagnostics.monitor.azure.com/;ApplicationId=xxxxxxxx-xxxx-xxxx-xxxx-xxxxxxxxxxxx". In our code we will create the constant, storing the connection string.
+
+```c#
+const string appInsightsConn = "InstrumentationKey=21e5a111-cd0c-4d22-963e-e9722648fff2;IngestionEndpoint=https://eastus-8.in.applicationinsights.azure.com/;LiveEndpoint=https://eastus.livediagnostics.monitor.azure.com/;ApplicationId=a298b1a1-2dc5-4bde-bd4a-1bbf6efec113";
+```
+
+To allow telemetry collection we will create the listeners, which will listen to Azure.AI.Inference activity and meters.In the code below we will create `tracerProvider`, which will listen to activity and `meterProvider`, listening to meter. We will add console exporters to both providers with the line `AddConsoleExporter()` and monitor exporter for Application Insights export: `AddAzureMonitorMetricExporter`. 
+
+```c#
+using var tracerProvider = Sdk.CreateTracerProviderBuilder()
+    .AddSource(ACTIVITY)
+    .ConfigureResource(r => r.AddService("MyServiceName"))
+    .AddConsoleExporter()
+    .AddAzureMonitorTraceExporter(options =>
+    {
+        options.ConnectionString = appInsightsConn;
+    })
+    .Build();
+
+using var meterProvider = Sdk.CreateMeterProviderBuilder()
+    .AddMeter(ACTIVITY)
+    .AddConsoleExporter()
+    .AddAzureMonitorMetricExporter(options =>
+    {
+        options.ConnectionString = appInsightsConn;
+    })
+    .Build();
+```
+
+Please note that if we will only create `tracerProvider` is required for both metrics and activity exports, but to export metrics we also need to define `meterProvider`.
```

Comment:
"if we only create `..` is required ..."

Seems like something is wrong with this the sentence.

Reply:
Fixed.